install/kubernetes/helm/istio/charts/sidecar-injector/templates/configmap.yaml

```diff
@@ -54,38 +54,42 @@ data:
         restartPolicy: Always
       containers:
       - name: istio-proxy
-        image: {{ .Values.global.hub }}/{{ .Values.global.proxy.image }}:{{ .Values.global.tag }}
+        image: {{ "[[ if (isset .ObjectMeta.Annotations \"sidecar.istio.io/proxyImageOverride\") -]]" }}
+        {{ "\"[[ index .ObjectMeta.Annotations \"sidecar.istio.io/proxyImageOverride\" ]]\"" }}
```

Comment:
Please also add a similar option for the initializer - there are already PRs doing fancier iptables.

Reply:
There is no initializer anymore. The mutating webhook and kube-inject (w/ `--injectConfigMapName`) both use this configmap.